core/DataTable.php

```diff
@@ -342,6 +342,17 @@ public function __destruct()
     }
 
     /**
+     * @ignore
+     * does not update the summary row!
+     */
+    public function setRows($rows)
```

Comment:
All the changes here and the new methods are needed for faster sorting in the `Sort` filter